### DISCUSSION TITLE
Loading option in splitButton

### DISCUSSION THREAD

**Discussion by jpeter06** [primefaces/community#1192] [PrimeNG] [answered]
As in _Button_, should be nice to have de **loading** and **loadingIcon** properties in _splitButton_.
This will allow  the user  notice that something it is happening and disable further clicks.

Thanks for this wonderfull UI Suite :-)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for the feedback, we'll wait for the community upvotes to implement this feature.

**Comment by jpeter06** [comment]
Any news about this option? Thanks for all
